### PR TITLE
done

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/react_people-table-advanced/#/

### CODE REVIEW

File: src/components/PeoplePage.tsx [hunk 1]
```diff
@@ -1,31 +1,133 @@
...
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  useEffect(() => {
+    async function fetchPerson() {
```
Comment (mykhalenych) [inline src/components/PeoplePage.tsx:26]: move from useEffect, and call only fetchPerson()

File: src/components/PeoplePage.tsx [hunk 2]
```diff
@@ -1,31 +1,133 @@
...
+      } finally {
+        setIsLoading(false);
+        setIsPeopleLoaded(true);
+      }
+    }
+
+    fetchPerson();
+  }, []);
+
+  let visiblePeople;
```
Comment (mykhalenych) [inline src/components/PeoplePage.tsx:46]:
```suggestion
const visiblePeople = order ? getPeopleShow().reverse() : getPeopleShow();
```
create after getPeopleShow

**Review by mykhalenych — CHANGES_REQUESTED** [review]

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
the search input doesn't work while the sex param is chosen
image

And try to pass all tests

**Review by DarkMistyRoom — APPROVED** [review]
you render people in reverse order from test expectations.
expectation:
image
your variant:
image

I approve the task, but you may fix it to pass the tests

**Pull request closed without merging**